read-the-docs-2021-stats.rst

```diff
@@ -0,0 +1,122 @@
+.. post:: Mar 17, 2022
+   :tags: stats, year-in-review
+
+Read the Docs 2021 Stats
+========================
+
+2021 continued in the realm of being a tough year.
+That said,
+Read the Docs had a lot of good things happen this year.
+
+We did a majority of the owrk on our :doc:`CZI grant </czi-grant-announcement>`,
+grew our team from 5 to 8,
+and continued to grow our `EthicalAds network`_.
+It's been a year of continued growth,
+and we hope to continue that into 2022.
+
+As we have for the past 8 years (!!),
+we are proud to share our 2021 stats post.
+
+.. _EthicalAds network: https://www.ethicalads.io/
+
+.. note::
+
+	You can always see our stats for the last `30 days`_.
+
+	Our posts from 2013_, 2014_, 2015_, 2016_, 2017_, 2018_, 2019_, and 2020_ are also available.
+
+.. _Read the Docs: https://readthedocs.org/
+.. _30 days: http://www.seethestats.com/site/readthedocs.org
+.. _2013: https://blog.readthedocs.com/read-the-docs-2013-stats/
+.. _2014: https://blog.readthedocs.com/read-the-docs-2014-stats/
+.. _2015: https://blog.readthedocs.com/read-the-docs-2015-stats/
+.. _2016: https://blog.readthedocs.com/read-the-docs-2016-stats/
+.. _2017: https://blog.readthedocs.com/read-the-docs-2017-stats/
+.. _2018: https://blog.readthedocs.com/read-the-docs-2018-stats/
+.. _2019: https://blog.readthedocs.com/read-the-docs-2019-stats/
+.. _2020: https://blog.readthedocs.com/read-the-docs-2020-stats/
+
+
+Page Views
+----------
+
+Our stats:
+
+* 700 Million Page Views (from 580 million)
+* 196 Million Unique Visitors (from 156 million)
+
+.. From Google Analytics
+
+These numbers are not completely accurate, since we :doc:`implemented Do Not Track support </do-not-track>` our analytics pageview figures understate our actual traffic.
```

Comment:
Slight run on

```suggestion
These numbers are not completely accurate. Since we :doc:`implemented Do Not Track support </do-not-track>` our analytics pageview figures understate our actual traffic.
```